osgp/protocol-adapter-oslp/oslp/src/main/java/org/opensmartgridplatform/oslp/OslpEncoder.java

```diff
@@ -7,17 +7,24 @@
  */
 package org.opensmartgridplatform.oslp;
 
-import org.jboss.netty.buffer.ChannelBuffer;
-import org.jboss.netty.buffer.ChannelBuffers;
-import org.jboss.netty.channel.Channel;
-import org.jboss.netty.channel.ChannelHandlerContext;
-import org.jboss.netty.handler.codec.oneone.OneToOneEncoder;
+import java.util.List;
 
-public class OslpEncoder extends OneToOneEncoder {
-    private static ChannelBuffer encodeMessage(OslpEnvelope envelope) {
-        int size = envelope.getSize();
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
-        ChannelBuffer buffer = ChannelBuffers.buffer(size);
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.handler.codec.MessageToMessageEncoder;;
+
+public class OslpEncoder extends MessageToMessageEncoder<OslpEnvelope> {
+
+    private static Logger LOGGER = LoggerFactory.getLogger(OslpEncoder.class);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make the "LOGGER" logger private static final. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1312)